packages/database/tests/degree/initialize.test.ts

```diff
@@ -1,13 +1,11 @@
 import { container, getSource } from '../../src/data-source'
 import { Degree } from '../../src/entity'
-import { DegreeRepository } from '../../src/repository'
 import Init from '../init'
-import { setup, teardown } from '../environment'
+import { setup, teardown, repo, t } from '../environment'
 import { Flatten, oneUp } from '../../src/utils'
 
 const dbName = oneUp(__filename)
 const db = getSource(dbName)
-const t: Partial<{ degree: Degree }> = {}
 
 beforeAll(() => setup(db))
 afterAll(() => teardown(db))
```

Comment:
`import { repo } from '../environment`

the variable `repo` will have loaded repositories ready-to-go after
`beforeAll(() => setup(db))`

Reply:
can consider splitting `repo` up to each of `ModuleRepository`, `UserRepository`, ... and importing only those needed in the test.